File: api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/utils/featuretoggling/Feature.java
package no.nav.sbl.dialogarena.modiabrukerdialog.api.utils.featuretoggling;

public enum Feature {

    DELVISE_SVAR("visDelviseSvarFunksjonalitet", false);

    public final String propertyKey;
    public final boolean defaultValue;

    Feature(String propertyKey, boolean defaultValue) {
        this.propertyKey = propertyKey;
        this.defaultValue = defaultValue;
    }

}

File: api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/utils/featuretoggling/Feature.kt
package no.nav.sbl.dialogarena.modiabrukerdialog.api.utils.featuretoggling

enum class Feature private constructor(val propertyKey: String, val defaultValue: Boolean) {

    SAMPLE_FEATURE("feature.samplerfeature", false),
    NYTT_VISITTKORT("modiabrukerdialog.nytt-visittkort", false),
    NY_BRUKERPROFIL("modiabrukerdialog.ny-brukerprofil", false),

}

